docs-src/concepts/what-is-an-action.md

```diff
@@ -0,0 +1,33 @@
+---
+id: what-is-an-action
+title: What is an Action?
+sidebar_label: Action
+description: An Action is the fundamental pricing unit in Temporal Cloud.
+tags:
+  - term
+  - explanation
+---
+
+An Action is the fundamental pricing unit in [Temporal Cloud](/concepts/what-is-temporal-cloud).
+
+**Workflows**
+
+- **Workflow started:** Occurs via client start, client Signal-With-Start, [Continue-As-New](/concepts/what-is-continue-as-new), [Child Workflow](/concepts/what-is-a-child-workflow-execution) start, or [Schedule](/concepts/what-is-a-schedule).
+  If a Workflow fails to start, an Action is not recorded.
+- **Workflow reset:** Actions that occurred before a [Reset](/concepts/what-is-a-reset) are counted even if they are no longer visible in [Event History](/concepts/what-is-an-event-history).
```

Comment:
I didn't realize "Workflow reset" was a type of action here, because the first bullet primed me to think:

- [context]: list of actions

and this feels like:

- [context]: no listed actions, just an FYI note.

I think my issue might be fixed if all bullets had a period > colon. Then I'm more likely to read as:

- [action]. explanatory text / notes

Reply:
Excellent point. I'll make that change.